lib/names/workloadendpoint_test.go

```diff
@@ -150,11 +150,6 @@ var _ = DescribeTable("WorkloadEndpoint name matching",
 		Pod:          "pod-name",
 		Endpoint:     "eth0",
 	}, "", false, "node"),
-	Entry("Orchestrator is missing", names.WorkloadEndpointIdentifiers{
```

Comment:
@robbrockbank how would you expect this code to work if everything but an orchestrator was provided? An error?